docs/ctk-B2-setup-steps.md

```diff
@@ -0,0 +1,148 @@
+# Setting up Compliance Tool Kit (CTK)
+
+## Pre-requisites
+
+CTK should be deployed with the required dockers.
+
+* compliance-toolkit-service: 0.0.9-B1
+* compliance-toolkit-ui: 0.0.9-B1
+
+Dependent Service (dockers)
+* Artifactory: mosipid/artifactory-ref-impl: 1.2.0.1-B2
+* Audit manager: mosipid/kernel-auditmanager-service: 1.2.0.1-B1
+* Auth Manager: mosipid/kernel-authmanager: 1.2.0.1-B1
+* Key Manager: modipid/kernel-keymanager-service: 1.2.0.1-B1
+* Partner Management: mosipid/partner-management-service: 1.2.0.1-B1
+* KeyCloak: mosipid/keycloak-init: 1.2.0.1-B1
+* Postgres: mosipid/postgres-init: 1.2.0.1-B1
+* Config Server: config-server: mosipid/config-server: 1.1.2
+* Notification Service: mosipid/kernel-notification-service: 1.2.0.1-B1
+* ClamAV: clamav/clamav: latest
+* MinIO
+
+_**Note**_: 
+* Ensure that in the `kernel-default.properties`, the value of `mosip-toolkit-client` is set as `auth.server.admin.allowed.audience`. 
+* If this was not set by default, then set it and restart `kernel-auth-service` and `compliance-toolkit-service`.
+* From 0.0.9-B2 version onwards, need to generate an encryption key for CTK.
+    * Create a new app id by directly inserting below row.
+	    
+		```
+			INSERT INTO keymgr.key_policy_def(app_id, key_validity_duration, is_active,pre_expire_days, 
+		   	access_allowed, cr_by, cr_dtimes, upd_by, upd_dtimes, is_deleted, del_dtimes)
+		   	VALUES ('COMPLIANCE_TOOLKIT', 1095, true, 60, 'NA', 'mosipadmin', '2022-11-28 09:00:40.822625', 
+		   	null, null, false, null);
+		
+		```
+	* Get the client token using auth manager swagger by calling endpoint.
+	    
+		`https://api-internal.dev.mosip.net/v1/authmanager/authenticate/clientidsecretkey`
+		
+		```jsonc
+		{
+			"id": "string",
+			"version": "string",
+			"requesttime": "2022-12-22T07:13:35.010Z",
+			"metadata": {},
+			"request": {
+				"clientId": " mosip-pms-client ",
+				"secretKey": " XXXXXX ",
+				"appId": " regproc "
+			}
+		}
+		```
+	
+	* Use `generateMasterKey` endpoint to generate module level certificate.
+	
+	    ![](_images/ctk-generateMasterKey.png)
+	
+	* Directly download certificate via keymanager swagger `getCertificate` with App Id as `COMPLIANCE_TOOLKIT` and Ref Id as `COMP-FIR`.
```

Comment:
Maintain consistency using app id.. is it app ID/ App Id/ app id ? should be same across our documents

Reply:
That is App Id